src/services/connector/serverless.yml

```diff
@@ -606,6 +606,62 @@ resources:
         Period: 60
         Statistic: Sum
         Threshold: 1
+    KafkaConnectServiceECSCpuAlarm:
+      Type: AWS::CloudWatch::Alarm
+      Properties:
+        AlarmName: ${self:service}-${sls:stage}-KafkaConnectService-CPUUtilization
+        AlarmDescription: "Trigger an alarm when the CPU utilization reaches 75%"
+        Namespace: AWS/ECS
+        MetricName: "CPUUtilization"
+        Dimensions:
+          - Name: ClusterName
+            Value: !Ref KafkaConnectCluster
+          - Name: ServiceName
+            Value:
+              !Select [
+                2,
+                !Split [
+                  "/",
+                  !Select [5, !Split [":", !Ref KafkaConnectService]],
+                ],
+              ]
+        Statistic: "Average"
+        Period: "300"
+        EvaluationPeriods: "2"
+        Threshold: "75"
+        ComparisonOperator: "GreaterThanOrEqualToThreshold"
```

Comment:
I don't think these needs to be in quotes. all of these values are strings so lets keep them consistent and not use quotes.

Reply:
and this also for the other places we are using quotes in this resource def

Reply:
quotes have been removed.